src/tests/ftest/get_remote_files.sh

```diff
@@ -371,6 +361,9 @@ if [ -n "${ARCHIVE_DEST}" ]; then
     if [ ${ret} -ne 0 ]; then
         rc=1
     fi
+
+    # Finally archive this script's log file
+    scp_files "${log_file}" "${ARCHIVE_DEST}"
```

Comment:
`${log_file}` only exists if `$VERBOSE` is true, yes?

Reply:
Yes, this should be conditional